playtron-os-files/playtron-os-files.spec

```diff
@@ -30,10 +30,11 @@ cp playtron-os-files-%{version}/LICENSE %{buildroot}/usr/share/licenses/playtron
 /usr/bin/clatd-ipv6-check
 /usr/bin/create-swap.sh
 /usr/bin/hwctl
+/usr/lib/modules-load.d/controllers.conf
+/usr/lib/os-release-playtron
 /usr/bin/playtron-factory-reset
 /usr/bin/resize-root-file-system.sh
 /usr/lib/sysctl.d/50-swappiness.conf
-/usr/lib/modules-load.d/controllers.conf
```

Comment:
Changed to align with alphabetical order.

Reply:
Beautiful. Thanks!